app/src/main/java/com/example/seizuredetectionapp/QuestionnaireMedical.java

```diff
@@ -49,58 +46,114 @@ protected void onCreate(Bundle savedInstanceState) {
 
         heightInches.setMinValue(0);
         heightInches.setMaxValue(11);
+        //heightInches.setTextColor(Integer.parseInt("black"));
 
+        openDatePicker.setOnClickListener(this);
         submitQuestionnaireMedical.setOnClickListener(this);
     }
 
     @Override
     public void onClick(View v) {
         switch (v.getId()) {
-            case R.id.submitQuestionnaireMedical:
-                saveQuestionnaireMedicalToFirebase();
-                break;
-        }
-    }
+            case R.id.openDatePickerDialog: {
+                DatePickerDialog datePickerDialog = new DatePickerDialog(
+                        this,
+                        0,
+                        this,
+                        Calendar.getInstance().get(Calendar.YEAR),
+                        Calendar.getInstance().get(Calendar.MONTH),
+                        Calendar.getInstance().get(Calendar.DAY_OF_MONTH)
+                );
+                datePickerDialog.show();
 
-    private void saveQuestionnaireMedicalToFirebase() {
-        String seizureDuration = String.valueOf(seizureDurationSeconds.getValue() + (seizureDurationMinutes.getValue() * 60));
-        String height = String.valueOf(heightInches.getValue() + (heightFeet.getValue() * 12));
-        String weight = weightInput.getText().toString().trim();
-        String seizureFrequencyPerMonth = seizureFrequency.getText().toString().trim();
-        String seizureStartD = seizureStartDate.getText().toString().trim();
-        String seizureStartM = seizureStartMonth.getSelectedItem().toString().trim();
-        String seizureStartY = seizureStartYear.getText().toString().trim();
-        String seizureT = seizureType.getSelectedItem().toString().trim();
-        String sex = sexInput.getSelectedItem().toString().trim();
-
-        //checks to see if any inputs are empty and alerts user.
-        if (height.equals("0")) {
-            heightFeet.requestFocus();
-            return;
-        }
+                break;
+            }
 
-        if (weight.equals("0")) {
-            weightInput.requestFocus();
-            return;
+            case R.id.submitQuestionnaireMedical: {
+                //Toast.makeText(QuestionnaireMedical.this, seizureStartD, Toast.LENGTH_LONG).show();
+                String seizureDuration = String.valueOf(seizureDurationSeconds.getValue() + (seizureDurationMinutes.getValue() * 60));
+                String height = String.valueOf(heightInches.getValue() + (heightFeet.getValue() * 12));
+                String weight = weightInput.getText().toString().trim();
+                String seizureFrequencyPerMonth = seizureFrequency.getText().toString().trim();
+                String seizureT = seizureType.getSelectedItem().toString().trim();
+                String sex = sexInput.getSelectedItem().toString().trim();
+
+                //checks to see if any inputs are empty and alerts user.
+                if (seizureDuration.equals("0")) {
```

Comment:
Make sure to give user an error when the height and weight are zero or seizure duration.